knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -748,3 +753,33 @@ func removeCreationTimestamp(manifest *mf.Manifest, _ *serverlessoperatorv1alpha
 	})
 	return err
 }
+
+func injectNamespacedBrokerMonitoring(apiClient client.Client) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "ConfigMap" || u.GetName() != "config-namespaced-broker-resources" {
+			return nil
+		}
+
+		eventingList := &operatorv1beta1.KnativeEventingList{}
+		err := apiClient.List(context.Background(), eventingList)
```

Comment:
With this additional logic, I think we need to watch KnativeEventing too here: https://github.com/openshift-knative/serverless-operator/blob/f1c6988f062e599cf16c210269e8d87c1479b542/knative-operator/pkg/controller/knativekafka/knativekafka_controller.go#L144-L146